app/assets/v2/js/activity.js

```diff
@@ -1,7 +1,7 @@
 /* eslint no-useless-concat: 0 */ // --> OFF
 window.addEventListener('load', function() {
-  setInterval(listen_for_web3_changes, 5000);
-  listen_for_web3_changes();
+  // setInterval(listen_for_web3_changes, 5000);
```

Comment:
cc @thelostone-mc who i know i've gone back and forth with on these issues.

Reply:
Ah so @octavioamu solution would be an on demand as opposed to always loading it.
My original PR -> help us identify all the parts where web3 was being used.

This would be easy to test as well cause we know which flows use this and test those out
Error rate would a almost none as compared to my PR cause that identifying the places where the actual listen_for_web3_changes would have to be made